httpd-client/tests/client.test.ts

```diff
@@ -0,0 +1,18 @@
+import { describe, expect, test } from "vitest";
+import { HttpdClient } from "../index";
+
+const client = new HttpdClient("http://0.0.0.0:8080");
+
+describe("client", () => {
+  test("#getRoot()", async () => {
+    await expect(client.getRoot()).resolves.not.toThrow();
```

Comment:
You can also write this as
```suggestion
    await client.getRoot()
```
This tests the same thing though the test failure message will be different.

Reply:
Yeah, you're right. I think I had it that way when I started out but then decided to have the `expect` there just so it looks like a spec. Reverted back: 8305cd53fee4cc2d56dd1e3387f421de358205b3.